scapy/consts.py

```diff
@@ -72,5 +74,11 @@ def test_pyx():
     uname = os.uname()
     LOOPBACK_NAME = "lo" if LINUX else "lo0"
 
+def getLoopbackInterface():
```

Comment:
Also, I don't like this because it adds a function call each time we need the interface. You should set the variable value once when `consts` is loaded and use it after.